### ISSUE TITLE
Need a New Release

### ISSUE THREAD

**Issue by zachcp** [CDK-R/cdkr#99]
rcdk is currently orphaned: https://cran.r-project.org/web/packages/rcdk/index.html.

This is my fault as I didn't update the correct Java version fast enough. in its current status it will start to break other people's submissions (dependent packages will fail). As per the official [R documentation](https://cran.r-project.org/doc/manuals/r-release/R-exts.html), we should be checking for Java Version using the following.

I can update and submit but I do not know if there is a special process for ORPHANED packages.

```R
.jinit()
jv <- .jcall("java/lang/System", "S", "getProperty", "java.runtime.version")
if(substr(jv, 1L, 2L) == "1.") {
  jvn <- as.numeric(paste0(strsplit(jv, "[.]")[[1L]][1:2], collapse = "."))
  if(jvn < 1.8) stop("Java >= 8 is needed for this package but not available")
}
```

**Comment by rajarshi** [comment]
Thanks for catching this as I've been pointing people to the latest code on Github. Definitely should put out a new release

**Comment by zachcp** [comment]
## Needed Before Resubmission to CRAN

- [x] add files needed for tests to `inst/molfiles`
- [x] use roxygen to document `bpdata`
- [x] add roxygen metadatafunctions for a large number of functions and update `Rd`
- [x] update for pkdgown and vignettes: `usethis:;use_vignette()` and `usethis::pkgdown()`
- [x] modify imports for S3 itertools to conform with new Roxygen (no @@S3 tag)
- [ ] Update Docs/params for a number of functions
- [ ] get vignettes to build and apply appropriate tests
- [ ] address any inconsistencies with previous Roxygen tests.
- [ ] version bump
- [ ] News.md/changelog

## Desired before Resubmisison to CRAN

- [ ] CI Testing (use travis)
- [ ] CI/CD build of pkgdown website (use travis +pkgdown)

**Comment by zachcp** [comment]
most of the document updates can be found in #101. If that looks good and is pulled i can write the news and bump and work on submission to CRAN. This needs to be  a simultaneous rCDKlibs and rCDK release due to new requirement for CDK >= 2.3

**Comment by rajarshi** [comment]
Just merged in the updates from #101 - all looks good.

**Comment by zachcp** [comment]
Final PR for Cran acceptance: https://cran.r-project.org/web/packages/rcdk/index.html
Current Package: https://cran.r-project.org/web/packages/rcdk/index.html
